### PR TITLE
Feat/adjust components for automated testing

### PR DESCRIPTION
**Pull request by YouQam**

### CODE REVIEW

File: src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.html [hunk 1]
```diff
@@ -83,17 +83,6 @@ <h2 hlmH2 class="tw-font-bold">2. {{ 'CreateBadge.enterDetails' | translate }}</
 				</div>
 			</div>
 
-			<p
```
Comment (jona159) [inline src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.html:86]: Why did you delete this?

Comment (YouQam) [reply src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.html]: I put it back.

**Review by jona159 — APPROVED** [review]
